Docs: Add recommendation for mac users while installing cms plugins (#1890)
vtex/faststore: apps/site/pages/docs/headless-cms-integration/1-configuring-the-vtex-account.mdx

![success-vtex-cms-sync](https://vtexhelp.vtexassets.com/assets/docs/src/faststore-vtex-cms___460365970fb39c94e300b9fbd497e17e.png)

<Callout type="warning" emoji="⚠️">

>   **Windows user**: if you find problems while installing

Author: ```suggestion
  <li>**Windows user**</li>: if you find problems while installing
```

  the plugin, please refer to the
  [Troubleshooting](/docs/headless-cms-troubleshooting/error-installing-headless-cms)
  article.

>   **macOS user**: if you find problems, run `yarn config set ignore-engines true` to ignore the node incompatibility versions while installing, then run `vtex plugins install cms` again.

Author: ```suggestion
  <li>**macOS user**</li>: if you find problems, run `yarn config set ignore-engines true` to ignore the node incompatibility versions, then run `vtex plugins install cms` again.
```

![success-vtex-cms-sync](https://vtexhelp.vtexassets.com/assets/docs/src/faststore-vtex-cms___460365970fb39c94e300b9fbd497e17e.png)

<Callout type="warning" emoji="⚠️">
  <li>**Windows user:** if you find problems while installing
  the plugin, please refer to the
  [Troubleshooting](/docs/headless-cms-troubleshooting/error-installing-headless-cms)
  article.</li> 
  <br></br>
  <li>**macOS user:** if you find problems, run `yarn config set ignore-engines true` to ignore the Node.js incompatibility version, then run `vtex plugins install cms` again. 

>   *Remember, you must have the latest Node.js version or a version higher than `15.0.0`.* </li> 

Reviewer: This command is to ignore the node version incompatibility so I think here should be smt like:
*Remember, it is recommended to use the latest Node.js version or a version higher than `15.0.0`.*

Author: Thanks for the feedback, Bibi <3

>   *Remember, you must have the latest Node.js version or a version higher than `15.0.0`.* </li> 

Author: ```suggestion
  *Remember, we recommend using the latest Node.js version or a version higher than `15.0.0`.* </li> 
```
